src/Commands/Modules/SoftModeration/EndorseCommand.cs

```diff
@@ -112,13 +112,15 @@ await builder.Build()
 
         [Command("topendorse"), Alias("topstar", "topstars")]
         [Summary("Show the users with most stars.")]
-        [Remarks("topendorse")]
+        [Remarks("topendorse [ignoreGuruStaff = true]")]
         [RequireGuru]
-        public async Task TopEndorseAsync() 
+        public async Task TopEndorseAsync(
+            [Summary("Whether or not to include Guru and Staff [default = false]")] bool defaultOnly = true) 

```

Comment:
Really don't wanna be nitpicky, but the `[]help` command extracts the parameter name, and currently the name displayed in the `Remarks` wouldn't match up with that. (`ignoreGuruStaff` vs. `defaultOnly`]